class/defaults.yml

```
@@ -1,4 +1,13 @@
 parameters:
   openshift4_operators:
     =_metadata: {}
-    namespace: syn-openshift4-operators
+    multi_instance: true
+    namespace: ${_instance}
+    defaultInstallPlanApproval: Automatic
+    defaultSourceNamespace: openshift-marketplace
+
+  openshift_operators:
+    defaultSource: community-operators
```

```suggestion
    defaultSource: certified-operators
```
certified-operators are probably the better default, because they come from the community but are verified, so probably more mature.